packages/testing/config/storybook/main.js

```diff
@@ -74,11 +74,12 @@ const baseConfig = {
       }
     }
 
-    const userPreviewPath = fs.existsSync(
-      redwoodProjectPaths.web.storybookPreviewConfig
-    )
-      ? redwoodProjectPaths.web.storybookPreviewConfig
-      : './preview.example.js'
+    let userPreviewPath = './preview.example.js'
+
+    if (fs.existsSync(redwoodProjectPaths.storybookPreviewConfig)) {
```

Comment:
I believe resolveFile will handle this. If the file doesn't exist, it the value will be null

```suggestion
    if (redwoodProjectPaths.storybookPreviewConfig) {
```